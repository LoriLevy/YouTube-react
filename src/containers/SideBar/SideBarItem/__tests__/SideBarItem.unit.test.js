import React from 'react';
import { shallow } from 'enzyme';
import SideBarItem from '../SideBarItem';

describe('SideBarItem', () => {
    test('Renders an EMPTY SideBarItem', () => {
        const wrapper = shallow(
            <SideBarItem />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders highlighted SideBarItem', () => {
        const wrapper = shallow(
            <SideBarItem highlighted icon='fire' label='Trending' />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders SideBarItem that is not highlighted', () => {
        const wrapper = shallow(
            <SideBarItem icon='fire' label='Trending' />
        );
        expect(wrapper).toMatchSnapshot();
    });
});