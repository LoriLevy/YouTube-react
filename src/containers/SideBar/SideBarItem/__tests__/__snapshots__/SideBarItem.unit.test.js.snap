// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarItem Renders an EMPTY SideBarItem 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        size="large"
      />
       
    </span>
    <span />
  </div>
</MenuItem>
`;

exports[`SideBarItem renders SideBarItem that is not highlighted 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        name="fire"
        size="large"
      />
       
    </span>
    <span>
      Trending
    </span>
  </div>
</MenuItem>
`;

exports[`SideBarItem renders highlighted SideBarItem 1`] = `
<MenuItem
  className="sidebar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        name="fire"
        size="large"
      />
       
    </span>
    <span>
      Trending
    </span>
  </div>
</MenuItem>
`;
